pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.black]
+line-length = 120
+skip-string-normalization = true
+target-version = ['py37', 'py38', 'py39']
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+)/
+'''
```

Comment:
What in the name of nonsense is this

Reply:
meh. An exclude list, to ensure black doesn't pick up uncessary folders with .py files. I'm reusing stuff I have around, this is probably not necessary. https://black.readthedocs.io/en/stable/usage_and_configuration/file_collection_and_discovery.html#gitignore